src/main/java/parser/GoCommandParser.java

```diff
@@ -1,42 +1,59 @@
 package parser;
 
-import commands.Command;
 import commands.GoChapterCommand;
 import commands.GoCommand;
 import commands.GoModuleCommand;
 import exception.IncorrectAccessLevelException;
 import exception.InvalidInputException;
 
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
 import static common.Messages.ADMIN;
-import static common.Messages.MESSAGE_ALPHANUMERIC_CHARACTERS;
-import static common.Messages.MESSAGE_MISSING_ARGS;
+import static common.Messages.CHAPTER;
+import static common.Messages.MESSAGE_INCORRECT_ACCESS_AT_CHAPTER_LEVEL;
+import static common.Messages.MESSAGE_MISSING_INDEX;
+import static common.Messages.MESSAGE_NON_INTEGER;
 import static common.Messages.MODULE;
 
 //@@author gua-guargia
 public class GoCommandParser {
-    private static final String MESSAGE_INCORRECT_ACCESS = "Go command can only be called at admin and module level.\n";
-
-    public Command parse(String commandArgs, String accessLevel)
+    public GoCommand parse(String commandArgs, String accessLevel)
```

Comment:
i think you can leave GoCommand as Command. but if its working then its fine

Reply:
Thank you! Have changed it to Command